README.md

```diff
@@ -20,18 +20,7 @@ gglegendcolour::changeLegendColour(
 ```
 Just create your ggplot2 graph as per normal, then use the changeLegendColour function with your ggplot object as the first argument and the second argument should be TRUE or FALSE depending on whether you wish to remove the key. The default value for the second argument is TRUE which removes the key from the returned ggplot2 object.
 
-In your ggplot2 object, you must supply colours to your groups. The vector you supply the colours in must be named with the names corresponding to the groups.
-
-If you wish to update the labels in your legend, as per the colours, the vector you supply the new labels in must be named with the names corresponding to the groups.
-
-As an example, if you're using scale_color_manual to supply your colours, this is what the values and labels arguments to that must look like:
-
-``` 
-scale_color_manual(
-  values = c("group 1" = "colour 1",...),
-  labels = c("group 1" = "colour 1",...)
-)
-```
+In your ggplot2 object, you must have a legend else you'll get an error. If you have multiple parts to the key in your legend, such as if your key had a line and a dot for each element in the key, the whole key for each element should be the same colour.
 
 # Example code
```

Comment:
I think the example code can be simplified (this example does work, but a much easier version works too!), just to show people it's nice and easy. E.g.:

```
library(ggplot2)

p <- ggplot(
  mtcars,
  aes(mpg, wt, colour = factor(cyl), fill = factor(cyl))
) +
  geom_point(shape = 21, alpha = 0.5, size = 2)

p <- gglegendcolour::changeLegendColour(
  x = p,
  remove_key = TRUE
)
p
```
also works nicely 😺

Reply:
Changed to your simplified example, also added it as an @example